src/lambda/explorer/lambdaNodes.ts

```diff
@@ -20,7 +20,7 @@ import { LambdaFunctionNode } from './lambdaFunctionNode'
 import { samLambdaImportableRuntimes } from '../models/samLambdaRuntime'
 
 export const CONTEXT_VALUE_LAMBDA_FUNCTION = 'awsRegionFunctionNode'
-export const CONTEXT_VALUE_LAMBDA_FUNCTION_IMPORTABLE = 'awsRegionFunctionNodeImportable'
+export const CONTEXT_VALUE_LAMBDA_FUNCTION_IMPORTABLE = 'awsRegionFunctionNodeDownloadable'
```

Comment:
```suggestion
export const CONTEXT_VALUE_LAMBDA_FUNCTION_DOWNLOADABLE = 'awsRegionFunctionNodeDownloadable'
```

Reply:
I'm guessing not changing this was intentional but eh doesn't matter too much to me.

Reply:
TBH I think it was a miss, I changed the value so it would make more sense in relation to the package.json file.